frontend/src/hooks/api/workspace.ts

```diff
@@ -46,7 +46,10 @@ export const useGetWorkspaceQuery = (workspaceSlug?: string) => {
 		};
 	}, [dispatch, query.data]);
 
-	return query;
+	const is404Error =
+		query.error && axios.isAxiosError(query.error) && query.error.response?.status === 404;
+
+	return { ...query, is404Error };
```

Comment:
This is just a suggestion.

How about move this logic to `axios` global configuration?
Then, we can handle all 404 errors in a single point.

Reference: https://velog.io/@pandati0710/axios-intercepter%EB%A1%9C-error-handling-%ED%95%98%EA%B8%B0

Reply:
I think it's a great idea! I will correct it right away.

Reply:
However, the request to move this to global logic is that when I read the code, each code was declared as axios. Do you want to distribute the common instance including 404 processing or 500 processing?

Reply:
Are there any potential issues that might arise when handling it with a global logic approach?

Reply:
Ah, there is no such problem, and I was a little worried about how to apply the code location.